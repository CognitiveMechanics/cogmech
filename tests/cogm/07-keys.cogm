k := "symbol"

a := <
  <#k, "keyed-symbol">,
  <k, "symbol">,
  <#"k", "keyed-literal">,
  <"k", "literal">
>

: #k
: #"k"

: a[#k]
: a[k]
: a[#"k"]
: a["k"]