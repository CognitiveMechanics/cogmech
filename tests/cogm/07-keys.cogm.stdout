<key, "symbol">
<key, "k">
"keyed-symbol"
"symbol"
"keyed-literal"
"literal"
